packages/network-console-shared/src/hosting/host-messages.ts

```diff
@@ -113,6 +113,7 @@ export interface IWebSocketConnectedMessage extends IMessage<'WEBSOCKET_CONNECTE
 
 export interface IWebSocketDisconnectedMessage extends IMessage<'WEBSOCKET_DISCONNECTED'> {
     requestId: string;
+    reason?: string;
```

Comment:
Can we please make sure this gets added to the Host Messages section of the protocol .md file?